src/scenes/Services/components/ServicesItem/components/List/List.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Slider from '@components/Slider';
+import Item from './components/Item';
+import { useDesktopStatus } from '@hooks';
+
+import styles from './List.module.scss';
+
+const List = ({ items }) => {
+  const { isDesktop } = useDesktopStatus(576);
```

Comment:
I don't think we should pass a number to this hook. It should have all breakpoints internally

Reply:
I like it. But what about API? Should argument be a string? For example: `useDesktopStatus('xs')` or `useDesktopStatus('md')` like it works in styles?

Anyway, I think we can leave opportunity to pass a special value. Just in case.